learn_script.py

```diff
@@ -4,39 +4,36 @@
 import shlex
 import subprocess
 import sys
-
+import argparse
 
 try:
     from ansimarkup import parse, ansiprint
 except:
-    ansiprint = lambda x: print(re.sub('<.*?>', '', x))
-
-
-script = open(sys.argv[1]).readlines()
+    def ansiprint(x): return print(re.sub('<.*?>', '', x))
```

Comment:
avoid def call on one-liners, specially on import definitions.

```suggestion
    ansiprint = lambda x: print(re.sub('<.*?>', '', x))
```

Reply:
This is black's fault, haha. I'll fix it.